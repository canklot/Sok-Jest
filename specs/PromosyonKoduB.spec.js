import TestComponent from '../components/Aktuelurunler.vue'
import { mount } from '@vue/test-utils'
import Vuetify from 'vuetify'

/* Vue.use(Vuetify) */

test('mount a vue component', () => {
    const wrapper = mount(TestComponent
    )
    expect(wrapper).toMatchSnapshot()
  })

/*   pass */