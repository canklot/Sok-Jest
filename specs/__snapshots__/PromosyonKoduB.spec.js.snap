// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount a vue component 1`] = `
<div class="Aktuel"><img src="../images/Sook.png" class="Sook"> <img src="../images/kettle2.png" class="kettle"> <img src="../images/TostMakinesi.png" class="TostMakinesi">
  <div class="Urun1"><span class="Yuvarlak"><p class="Garanti">	2 YIL GARANTİLİ</p></span>
    <div class="Marka">Ben defaultum TEBA</div>
    <div class="UrunAdi">Ben defaultum </div>
    <div class="Fiyat">₺99</div>
    <div class="Aciklama">1,8lt/ Paslanmaz çelik rezistans Su kaynadığında otomatik kapanma/ Susuz kullanılmaya karşı koruma sistemi</div>
  </div>
  <div class="Urun2"><span class="Yuvarlak2"><p class="Garanti">	2 YIL GARANTİLİ</p></span>
    <div class="Marka">TEBA</div>
    <div class="UrunAdi">ELEKTRİKLİ CAM SU ISITICI</div>
    <div class="Fiyat">₺89,90</div>
    <div class="Aciklama">1,8lt/ Paslanmaz çelik rezistans Su kaynadığında otomatik kapanma/ Susuz kullanılmaya karşı koruma sistemi</div>
  </div>
</div>
`;
